node/test/Test/Pos/CborSpec.hs

```diff
@@ -223,11 +232,12 @@ roundtripProperty (input :: a) = ((deserialize . serialize $ input) :: a) === in
 -- without breaking anything. This should work with every time which adopted
 -- the schema of having at least one constructor of the form:
 -- .... | Unknown Word8 ByteString
-extensionProperty :: forall a. (Arbitrary a, Eq a, Show a, Bi a) => Property
-extensionProperty = forAll @a (arbitrary :: Gen a) $ \input ->
+extensionPropertyOn :: forall a b. (Eq a, Show a, Bi a, Arbitrary b, Eq b, Show b, Bi b)
+                    => Property
+extensionPropertyOn = forAll @b (arbitrary :: Gen b) $ \input ->
```

Comment:
Do we need to use `forAll arbitrary $ \ .. -> ...` here? Isn't this exactly the same as using

    extensionPropertyOn :: (Eq a, Bi a, Eq b, Bi b) => Proxy a -> b -> Bool
    extensionPropertyOn _ input =